Scripts/swiftlint-output.sh

```diff
@@ -0,0 +1,9 @@
+if which swiftlint >/dev/null; then
+  # Lint generated code in Tests/Expected
+  for f in `find "Tests/Expected" -name '*.swift'`
+  do
+    cat $f | swiftlint lint --use-stdin | sed s:'<nopath>':"$f":
```

Comment:
I remember that I had to do this dance with `stdin` because `swiftlint` didn't accept files not ending in `*.swift`
So basically if I did `swiftlint lint Tests/Expected/foo.swift.out` then `swiftlint` would ignore the input file (because it didn't have a `.swift` extension), hence the `cat` + `--use-stdin` trick… + the replacement of `<nopath>` then.

Now that the files in `Tests/Expected` have been renamed to `*.swift` we should be able to directly use `swiftlint lint $f` I think. Or maybe even use `find "Tests/Expected" -name '*.swift' -0 | xcargs -0 swiftlint lint`?